src/hooks/useNotifications.ts

```diff
@@ -79,15 +79,13 @@ export const useNotifications = (): NotificationsState => {
   const fetchNotifications = useCallback(
     async (accounts: AuthState, settings: SettingsState) => {
       function getGitHubNotifications() {
-        if (!isGitHubLoggedIn(accounts)) {
-          return;
+        if (isGitHubLoggedIn(accounts)) {
+          return listNotificationsForAuthenticatedUser(
+            Constants.DEFAULT_AUTH_OPTIONS.hostname,
+            accounts.token,
+            settings,
+          );
```

Comment:
I'm not sure about this change. We had a explicit return before, and now it is implicit, even though we get cleaner code... What do you think?

Reply:
Great feedback.  I've reverted this section back to it's previous explicit early return form.